.github/workflows/ci.yml

```diff
@@ -70,13 +64,19 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      # Install the Rust toolchain for RISC-V devices:
+      # Install the Rust stable and nightly toolchains for RISC-V devices:
```

Comment:
This appears to just install both toolchains, and not build with both of them? Or am I missing something here?